recipes/altair-aitools-devkit/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "altair-aitools-devkit" %}
+{% set version = "1.0b1" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
Please read the documentation in the link above.

Reply:
Are you referring to the use of the `rc` tag instead of the `beta` tag? or am I supposed to create a conda_build_config.yaml? because the doc says that a pre-release version must change the aforementioned file, but I did not find any guidance on what this file should contain... 😕

Reply:
> Are you referring to the use of the `rc` tag instead of the `beta` tag?

Both are subject to CFEP-05.

> or am I supposed to create a conda_build_config.yaml? because the doc says that a pre-release version must change the aforementioned file, but I did not find any guidance on what this file should contain... 😕

Correct. See https://github.com/conda-forge/cfep/blob/main/cfep-05.md#specification

You need to create a `conda_build_config.yaml` inside your recipe directory and add:

```yaml
channel_targets:
  - conda-forge altair-aitools-devkit_dev
```

to it.

Reply:
all right, I understand it now. my concern was that the docs did not explicitly say that this is the only thing needed to be added to the file and the term `modify` in my book left it unclear whether this is the only content of the file.